src/components/ChecOption.vue

```diff
@@ -89,19 +102,25 @@ export default {
 .option {
   @apply w-full flex items-center text-sm px-4 py-3 outline-none cursor-pointer bg-white;
 
-  &:hover {
+  &:hover:not(.option--loading) {
     @apply bg-gray-100;
   }
```

Comment:
```suggestion
  &:hover {
    &:not(.option--loading) {
      @apply bg-gray-100;
    }
  }
```

Reply:
Why?

Reply:
Thought id suggest since we're doing it within the :active, and :focus states, although we have separate styles within those rules. Can ignore it though 🙈